README.md

```diff
@@ -28,7 +28,10 @@ We try to solve this problem by adding a Gateway which can be considered the ent
 | --- | --- | --- |
 | `-gateway.distributor.address` | Upstream HTTP URL for Cortex Distributor | (empty string) |
 | `-gateway.query-frontend.address` | Upstream HTTP URL for Cortex Query Frontend | (empty string) |
-| `-gateway.auth.jwt-secret` | HMAC secret to sign JSON Web Tokens | (empty string) |
+| `-gateway.auth.jwt-secret` | HMAC secret or RSA public key to verify JSON Web Tokens | (empty string) |
```

Comment:
It's a good practice to rotate RSA public keys and with such approach it can't be done easily. Would be much better to get them from external url, cache for period of time and use for JWT verification.